Lib/test/_test_multiprocessing.py

```diff
@@ -2712,6 +2712,27 @@ def errback(exc):
         p.close()
         p.join()
 
+    def test_pool_worker_died_without_communicating(self):
+        # Issue22393: test fix of indefinite hang caused by worker processes
+        # exiting abruptly (such as via os._exit()) without communicating
+        # back to the pool at all.
+        prog = (
```

Comment:
This can be written much more clearly using a multi-line string. See for example a very similar case in `test_shared_memory_cleaned_after_process_termination` in this file.